tcms/testruns/static/testruns/js/get.js

```diff
@@ -33,8 +33,41 @@ $(document).ready(() => {
         }
     });
 
-    $('.add-hyperlink-bulk').click(function() {
+    $('.add-comment-bulk').click(() => {
+        const selected = selectedCheckboxes()
```

Comment:
Doesn't close the drop down menu, see how other bulk actions do it, e.g. line 69